docs/runbooks/KubevirtHyperconvergedClusterOperatorNMOInUseAlert.md

```diff
@@ -1,43 +1,59 @@
 # KubevirtHyperconvergedClusterOperatorNMOInUseAlert
+<!--apinnick Nov 2022-->
 
 ## Meaning
-Starting from version 4.11.0, Node Maintenance Operator (NMO) is no longer included in OpenShift Virtualization.  
-This alert is being fired if both of the following conditions are met:
-- OpenShift version is 4.11 or higher.
-- The integrated NMO is being used, i.e. one or more custom resources of Kind `NodeMaintenance` and API group of `nodemaintenance.kubevirt.io` have been found on the cluster.
 
-Existence of such a custom resource indicates that the targeted node specified in `spec.nodeName` was put into a maintenance mode, which means the node is [cordoned](https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#cordon) and server-side [drained](https://kubernetes.io/docs/tasks/administer-cluster/safely-drain-node/#use-kubectl-drain-to-remove-a-node-from-service).
+This alert fires when integrated Node Maintenance Operator (NMO) custom resources (CRs) are detected on OpenShift Virtualization 4.10.
```

Comment:
I changed the version here to "4.10" because "4.11" does not make sense if the alert blocks upgrade to 4.11.

Reply:
This alert was removed in 4.11. So 4.11+ user should never see it. So the change to 4.10 seems correct to me. However, I would maybe change the sentence to reflect that.

Reply:
@nunnatsa Do you mean to also add that this alerts is only relevant to 4.10 installations?